addons/mail/static/src/core/common/composer_model.js

```diff
@@ -8,6 +9,7 @@ export class Composer extends Record {
         this.attachments.length = 0;
         this.replyToMessage = undefined;
         this.text = "";
+        this.body = markup("<p><br></p>");
```

Comment:
isn't this useless/not used in the context of this pr?

Reply:
Yes, let's just introduce only useful/functional code. Fields for which we never read the value have no reason being added until at least one place is reading them.